hivemind/optim/experimental/state_averager.py

```diff
@@ -100,7 +100,7 @@ def __init__(
         self.offload_optimizer = offload_optimizer
         self.custom_gradients = custom_gradients
 
-        self._main_parameters, self._parameter_names = main_parameters, parameter_names
+        self.main_parameters, self.parameter_names = main_parameters, parameter_names
```

Comment:
made them public because these params are needed in hivemind.Optimizer.step

... and they are no more private than, for instance, opt_keys_for_averaging